src/transformers/training_args.py

```diff
@@ -236,6 +236,8 @@ class TrainingArguments:
             Random seed to be used with data samplers. If not set, random generators for data sampling will use the
             same seed as `seed`. This can be used to ensure reproducibility of data sampling, independent of the model
             seed.
+        jit_mode(`bool`, *optional*, defaults to `False`):
```

Comment:
We should maybe add an `eval` suffix here to make it clear to the user this is inference only.

Reply:
sure, have made changes, thanks for the suggestion.